template_registry.go

```diff
@@ -169,10 +179,14 @@ func (d *TemplateRegistryDir) Get(path []string) (t *Template, err error) {
 		return nil, fmt.Errorf("wrong path \"%s\", should have at least one chunk", strings.Join(path, "/"))
 	}
 	if len(path) == 1 {
-		t, exists := d.content[path[0]]
+		i, exists := d.content[path[0]]
```

Comment:
I guess we can replace it with ``switch-case``

```Golang
m := map[string]interface{}{
    "string":"string",
    "nil":nil,
    "number":5,
  }
  k := "wrongKey"
  switch v := m[k].(type) {
    case string:
      fmt.Printf("\n%s is string \"%s\"", k, v)
    case nil:
      fmt.Printf("\n%s is nil", k)
    default:
      fmt.Printf("\n%s is something else", k)
    }
```
    
 will print:
    
`wrongKey is nil`